provider/datagen/src/crabbake.rs

```diff
@@ -133,8 +138,10 @@ impl DataExporter<CrabbakeMarker> for ConstExporter {
                         string
                     })
                     .reduce(|mut a, b| {
-                        a.push('_');
-                        a.push_str(&b);
+                        if a.len() < 35 {
```

Comment:
question: why? / please comment

Reply:
Otherwise calendars will have identifiers like `AR_U_CA_BUDDHIST_AR_U_CA_COPTIC_AR_U_CA_INDIAN_AR_U_CA_JAPANESE_AR_EG_U_CA_BUDDHIST_AR_EG_U_CA_COPTIC_AR_EG_U_CA_INDIAN_AR_EG_U_CA_JAPANESE` wich is a bit much